src/style/flatpickr.styl

```diff
@@ -245,6 +246,7 @@
 
     svg
         width 14px
+        height 14px
```

Comment:
I haven't noticed any issues with svgs in IE. I'm guessing this fixes a case where they appear huge? How do I reproduce it?

Reply:
Theyappear too small in Edge / IE11 as it needs both values as preserveAspect is not used snd the size calculated incorrectly. May be native or in combination with svg4everybody but IE needs both values for right calculations of svg elements.